core/domain/subtopic_page_services_test.py

```diff
@@ -117,3 +164,12 @@ def test_commit_log_entry(self):
             subtopic_page_commit_log_entry.subtopic_page_id,
             self.subtopic_page_id)
         self.assertEqual(subtopic_page_commit_log_entry.user_id, self.user_id)
+
+    def test_delete_subtopic_page(self):
```

Comment:
Try also deleting subtopics that don't exist (slightly different from delete existing, and then try deleting again)

Reply:
Added the test for it.